lint-staged.config.js

```diff
@@ -1,11 +1,8 @@
 module.exports = {
-  "*.{js,ts}": [
-    "prettier --write",
-    'eslint --ignore-pattern "**/build-scripts/**/*.js" --fix',
-  ],
+  "*.{js,ts}": ["prettier --write", "eslint --fix"],
   "!(/translations)*.{json,css,md,html}": "prettier --write",
   "translations/*/*.json": (files) =>
-    'printf "%s\n" "These files should not be modified.  Instead, make the necessary modifications in src/translations/en.json.  Please see translations/README.md for details." ' +
+    'printf "%s\n" "Translation files should not be added or modified here.Instead, make the necessary modifications in src/translations/en.json.  Other languages are managed externally. Please see https://developers.home-assistant.io/docs/translations/ for details." ' +
```

Comment:
```suggestion
    'printf "%s\n" "Translation files should not be added or modified here. Instead, make the necessary modifications in src/translations/en.json. Other languages are managed externally. Please see https://developers.home-assistant.io/docs/translations/ for details." ' +
```

Reply:
What's the change?  My screen reader and I are missing it.

Reply:
missing and double spaces, just minor things :-)